molr-mole-server/src/main/java/io/molr/mole/server/rest/MolrMoleRestService.java

```diff
@@ -36,7 +35,7 @@ public MolrMoleRestService(Mole mole) {
         GET mappings
      */
 
-    @GetMapping(path = "/states", produces = MediaType.APPLICATION_STREAM_JSON_VALUE)
+    @GetMapping(path = "/states", produces = MediaType.TEXT_EVENT_STREAM_VALUE)
```

Comment:
This is more a general comment (as it was already the same than in the previous version), I think we could consider having a static import to `MediaType.TEXT_EVENT_STREAM_VALUE` (and similar), as we also do in the client ... (any objections from the others?)

Reply:
Agree :)